paparazzi/settings.gradle

```diff
@@ -1,5 +1,3 @@
-rootProject.name = 'paparazzi-libs'
```

Comment:
Looks like it gets confused on the root project and the subproject both being called paparazzi. IDEA has had a long issue where included builds are misconfigured and don't work as well. I think you might be experiencing that issue. Which version of AS/IJ are you using?

Reply:
IntelliJ IDEA 2022.3.1 (Community Edition)
Build #IC-223.8214.52, built on December 20, 2022
Runtime version: 17.0.5+1-b653.23 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 13.1
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 8
Metal Rendering is ON
Registry:
    debugger.new.tool.window.layout=true
    ide.experimental.ui=true

Non-Bundled Plugins:
    IdeaVIM (2.0.0)

Kotlin: 223-1.7.21-release-272-IJ8214.52

Reply:
Android Studio Flamingo | 2022.2.1 Canary 10
Build #AI-222.4459.24.2221.9409768, built on December 15, 2022
Runtime version: 17.0.4.1+0-17.0.4.1b469.62-9127311 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 13.1
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 8
Metal Rendering is ON
Registry:
    external.system.auto.import.disabled=true
    ide.text.editor.with.preview.show.floating.toolbar=false
    gradle.version.catalogs.dynamic.support=true

Non-Bundled Plugins:
    IdeaVIM (1.11.1)

Reply:
Hmm, these are way too new, unless the issue regressed. It's interesting that the compilation fails as it does. I think that might be a Gradle bug:
repro: name root project same as one of the submodules and enable project accessors. I'll try to repro and report.

Reply:
I think maybe we can take a step back. I have a feeling it'll work even without a code change. Are you importing the repository root? If so, try importing only the paparazzi subfolder. Try this on master.

Reply:
Perhaps I should create a separate issue that demonstrates the problem I'm trying to address. Essentially, without these changes, when you import the project and attempt to run a test using context actions from a file or from a test directory, the generated configuration will contain `paparazzi-libs` as the root task name instead of `paparazzi`. After these changes, I am able to run tests just fine from the AS and IntelliJ

Reply:
That makes sense, but that sounds like a bug in the IJ core (create issue in youtrack.com for sure):
Based on OP and your description above IJ confuses the `rootProject.name` and the included build's name. It's trying to be too smart and uses the included build's `rootProject.name` rather than the name that it actually has in at the point of include.

What you're trying to do is a workaround for this bug, but it's not compatible with Gradle. I'm surprised that it works, I'll have a go when I get back after holidays.

What I proposed above is another workaround: rather than importing "repo/" in IJ, import "repo/paparazzi/" folder. At this point your only editing the "libs" project and it should be listed exactly as is, because there's no included build in play. My base assumption around included build might be wrong.

I'm curious if everyone experiences what you're seeing. Definitely have a look at YouTrack as well because this behaviour sounds buggy.